docs/source/package.rst

```diff
@@ -1312,6 +1312,20 @@ Below is a list of compatible elements.
 +-----------------------------------------+----------------------------------------------------------+
 | ``abstract``/``description``            | ``doc``                                                  |
 +-----------------------------------------+----------------------------------------------------------+
+| ``version``                             | ``s:version``/``s:softwareVersion``                      |
++-----------------------------------------+----------------------------------------------------------+
+
+Additional compatible elements found under ``metadata`` in :term:`WPS` Context
+
+Parameters in :term:`CWL` Context:
+
+- ``s:author``
+- ``s:citation``
+- ``s:codeRepository``
+- ``s:contributor``
+- ``s:dateCreated``
+- ``s:license``
+- ``s:releaseNotes``
```

Comment:
For this, I believe it should be added directly to the ``metadata`` line that is already present in the table. Basically, just extend the right side (where ``$schemas``/``$namespace`` is indicated) with the mention of those fields. 

You can also add the `[#cwl_schemaorg]_` footnote reference (as in the ``s:keywords`` case), that points to the CWL metadata example below, which contains all of those fields.